src/commands/service/create.test.ts

```diff
@@ -25,38 +26,135 @@ afterAll(() => {
   disableVerboseLogging();
 });
 
-describe("validate pipeline config", () => {
-  const configValues: any[] = [
-    "testHelmChart",
-    "testHelmRepo",
-    "testHelmConfigBranch",
-    "testHelmConfigGit",
-    "/test/path",
-    "testService",
-    "test/packages",
-    "test-maintainer",
-    "test@maintainer.com",
-    "my,middleware,string",
-    true,
-    "testDisplayName",
-    80,
-    "pathPrefix",
-    "version",
-    "backend"
-  ];
-
-  it("config is valid", () => {
-    expect(isValidConfig.apply(undefined, configValues as any)).toBe(true);
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+const mockValues: ICommandValues = {
+  displayName: "",
+  gitPush: false,
+  helmChartChart: "",
+  helmChartRepository: "",
+  helmConfigBranch: "",
+  helmConfigGit: "",
+  helmConfigPath: "",
+  k8sBackend: "",
+  k8sBackendPort: "80",
+  k8sPort: 80,
+  maintainerEmail: "",
+  maintainerName: "",
+  middlewares: "",
+  middlewaresArray: [],
+  packagesDir: "",
+  pathPrefix: "",
+  pathPrefixMajorVersion: "",
+  variableGroups: []
+};
+
+const getMockValues = (): ICommandValues => {
+  // TOFIX: if possible, can we use lodash?
```

Comment:
You can create an issue to be prioritized and delete the comment.

Reply:
ok

Reply:
https://github.com/microsoft/bedrock/issues/933